torch/export/exported_program.py

```diff
@@ -413,7 +420,11 @@ def _decompose_and_get_gm_with_new_signature_constants(
                                     entry
                                 ]
 
-        with fake_mode:
+        with fake_mode, _override_composite_implicit_decomp(
+            # preserve understanding it is not functional, replace with aten._to_copy later
+            (torch.ops.aten.to.dtype_layout,),
```

Comment:
This is more of a question to @bdhirsh and @zou3519: We are basically disabling CIA decomp for aten.to here. By the time, we hit the post-dispatch FunctionalTensor mode in python, do we guarantee that it has never been went through C++ functionalization decomp? Do we always make sure we hit the PythonKey before hitting Functionalize key in post-dispatch world? I think previously it didn't matter because we only preserved functiuonal CIA ops anyways.